TwilioStore.Tests/Services/NotificationServiceTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using TwilioStore.Interfaces.Exceptions;
+using TwilioStore.Interfaces.Services;
+using TwilioStore.Services;
+
+namespace TwilioStore.Tests.Services
+{
+    [TestClass]
+    public class NotificationServiceTest
+    {
+        // Get your test credentials from https://www.twilio.com/console/account/settings
+        private const string TestAccountSid = "ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx";
+        private const string TestAuthToken = "0123456789abcdef0123456789abcdef";
+
+        // Magic numbers from https://www.twilio.com/docs/api/rest/test-credentials
+        private const string ValidFromNumber = "+15005550006";
+        private const string ValidToNumber = "+18778894546";
+        private const string InvalidNumber = "+15005550001";
+        private const string AvailableNumber = "+15005550006";
+        private const string UnavailableNumber = "+15005550000";
+
+        private static INotificationConfiguration GetTestConfig()
```

Comment:
I think we can remove `GetTestConfig` by adding a _app.config_ file, which will contain the test credentials. I stole this trick from the Java world.

```
  <appSettings>
    <add key="TwilioAccountSid" value="ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" />
    <add key="TwilioAuthToken" value="0123456789abcdef0123456789abcdef" />
    <add key="TwilioFromNumber" value="15005550006" />
  </appSettings>
```

If we decide to use this approach we simplify the `NotificationService` constructor to use only the parameterless version. What do you think?